esp/src/eclwatch/LFDetailsWidget.js

```diff
@@ -425,8 +425,6 @@ define([
                 dom.byId(this.id + "ProtectedImage").src = this.logicalFile.getProtectedImage();
             } else if (name === "IsCompressed") {
                 dom.byId(this.id + "CompressedImage").src = this.logicalFile.getCompressedImage();
-            } else if (name === "IsProtected") {
-                this.updateInput("isProtected", oldValue, newValue);
```

Comment:
@kunalaswani - I think this logic is needed to set the correct state of the checkbox on load and on refresh?